packages/modelserver-plugin-ext/src/executor.ts

```diff
@@ -33,10 +33,11 @@ export interface Executor {
      * Apply a JSON patch to the model.
      *
      * @param patch the JSON patch to apply
+     * @param format optional format parameter to encode the returned update result, defaults to `json-v2`
```

Comment:
Should this not also indicate that the `patch` argument is encoded in the specified `format`? That is generally how all of the `ModelServerClient` APIs work: the `format` covers both inputs and outputs of the API. It's confusing that this applies only in one direction.

Reply:
Yes I agree, I was a bit distracted about the initial usage of the `json` format, but now after a bit of investigation it is clear that the json patch format (`json-v2`) has to be applied also for the received update via patch.